proto/sequencers/sequencers.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+package rollapp.sequencers.types;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+import "cosmos/staking/v1beta1/staking.proto";
+
+option go_package = "github.com/dymensionxyz/dymension-rdk/x/sequencers/types";
+
+// Params defines the parameters for the module.
+message Params {
+  option (gogoproto.equal) = true;
+  option (gogoproto.goproto_stringer) = false;
+
+  // UnbondingTime is the time duration of unbonding.
+  google.protobuf.Duration unbonding_time = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.stdduration) = true,
+    (gogoproto.moretags) = "yaml:\"unbonding_time\""
+  ];
+
+  // HistoricalEntries is the number of historical entries to persist.
+  uint32 historical_entries = 2
+      [ (gogoproto.moretags) = "yaml:\"historical_entries\"" ];
+}
+
+message Sequencer {
+  // Validator is a convenient storage for e.g operator address and consensus pub key
+  cosmos.staking.v1beta1.Validator validator = 1;
+  // RewardAddr is the sdk acc address where the sequencer has opted to receive rewards. Empty if not set.
+  string reward_addr = 2;
+  // Relayers is an array of the whitelisted relayer addresses. Addresses are bech32-encoded strings.
+  repeated string relayers = 3;
```

Comment:
seems like this is repeated, and also the `WhiteListedRelayers` string is repeated? so list of lists? why is that?